lib/maths/CBoostedTreeImpl.cc

```diff
@@ -485,6 +493,12 @@ void CBoostedTreeImpl::train(core::CDataFrame& frame,
         m_BestForest = this->trainForest(frame, this->allTrainingRowsMask(), recordMemoryUsage);
         this->recordState(recordTrainStateCallback);
 
+        timeAccumulator.add(stopWatch.stop());
+
+        LOG_INFO(<< "Training finished after " << m_CurrentRound << " iterations. Time per iteration in ms mean: "
+                 << timeAccumulator.s_Moments[0]
+                 << " std. dev:  " << std::sqrt(timeAccumulator.s_Moments[1]));
```

Comment:
This is the unbiased variance estimate and what we'd normally use
```suggestion
                 << " std. dev:  " << std::sqrt(CBasicStatistics::variance(timeAccumulator)));
```